modules/utxo-lib/test/integration_local_rpc/compare.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * @prettier
+ */
+import { RpcTransaction } from './generate/RpcTypes';
+import * as networks from '../../src/networks';
+import { Network } from '../../src/networkTypes';
+import { getMainnet, isZcash } from '../../src/coins';
+
+type NormalizedObject = Record<string, unknown>;
```

Comment:
Is there a reason we can't make this more specific? It seems like we should be able to define the fields we should include, and I think it'd make the `normalize` functions below easier to follow.

Reply:
The point of the tests is actually to discover new properties that might pop up in either the RPC response or the parsed object and deal with them as they appear.

If I exhaustively enumerated the tx properties, it would suggest that we know them upfront, when often we do not.   

Let's assume one of our coins makes a protocol upgrade and a new field `foo` appears: these tests will fail unless we _decide_ that either `foo` is not important (we delete it) or it needs to be parsed and exposed. 

If we were only comparing know properties that would not work.

Reply:
Thanks, I thought that might be the case but wasn't clear on it. Then this approach all makes sense.